resources/js/Components/ComponentsPraktikans/PraktikanCard.jsx

```diff
@@ -0,0 +1,53 @@
+import daskomIcon from "../../../../resources/assets/daskom.svg";
+import CircularImage from "../CircularImage";
+
+export default function CardPraktikan() {
+
+    //data variable
+    let name = "Dhea Aisyah Putri";
+    let nim = "1104223146";
+    let kelas = "TF-46-04";
+    let notel = "081250770473";
+    let email = "dheaaisyahputri09@gmail.com";
+    let major = "S1 Teknik Fisika";
```

Comment:
Please never put any real data or [PII](https://en.wikipedia.org/wiki/Personal_data) anywhere in the codebase 😅

Reply:
ah I forgot to change it, my bad